pkgs/test_core/lib/src/runner.dart

```diff
@@ -298,6 +299,30 @@ class Runner {
             return false;
           }
 
+          // Skip tests that don't start on `line` or `col` if specified.
+          var line = suite.config.line;
+          var col = suite.config.col;
+          var trace = test.trace;
+          if (trace == null && (line != null || col != null)) {
+            return false;
+          } else if (trace != null) {
+            var absoluteSuitePath = p.absolute(suite.path!);
+            if (line != null &&
+                !trace.frames.any((frame) =>
+                    frame.uri.scheme == 'file' &&
+                    frame.uri.toFilePath() == absoluteSuitePath &&
+                    frame.line == line)) {
+              return false;
+            }
+            if (col != null &&
+                !trace.frames.any((frame) =>
```

Comment:
Synced offline about enforcing line and column match the same frame, not any frame at all.